instagram-page-react/src/pages/profilePage/organisms/posts/reducers/postListReducer.js

```diff
@@ -0,0 +1,23 @@
+// Lodash
+import _get from 'lodash/get';
+
+// Constants
+import {EMPTY_OBJECT} from "../../../../../app.constants";
+
+import ACTION_TYPES from "./actionTypes";
+
+export const INITIAL_STATE = {
+    POSTS: EMPTY_OBJECT,
+}
+
+const postListReducer = (state = INITIAL_STATE, action) => {
+    if(action.type === ACTION_TYPES.SET_POST){
+        return {
```

Comment:
This breaks SRP and when we have multiple if checks it is preferable to create a map of handlers
We can use `handleActions` from the `redux-actions` library